docs/examples/module_step2.py

```diff
@@ -26,9 +26,6 @@ async def get_readings(self, extra: Optional[Dict[str, Any]] = None, **kwargs) -
         wifi_signal = [x for x in content[2].split(" ") if x != ""]
         return {"link": wifi_signal[2], "level": wifi_signal[3], "noise": wifi_signal[4]}
 
-    async def get_geometries(self, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None) -> List[Geometry]:
```

Comment:
Does every `Sensor` need to define `get_geometries()` or is it unnecessary?

Reply:
it's unnecessary because the `ComponentBase` parent class already implements it as unimplemented